### ISSUE TITLE
`auro-tabs` remove hyperlink root component

### ISSUE THREAD

**Issue by DukeFerdinand** [AlaskaAirlines/auro-tabs#51]
### Is your feature request related to a problem? Please describe.

Use a div or non-focusable wrapper as nested focusable entities are a direct accessibility violation
